plugins/SitesManager/templates/siteWithoutData.twig

```diff
@@ -89,6 +89,11 @@
             <a href="{{ linkTo({module: 'SitesManager', action: 'ignoreNoDataMessage'}) }}"
                class="btn ignoreSitesWithoutData">{{ 'SitesManager_SiteWithoutDataIgnoreMessage'|translate }}</a>
 
+            <a class="btn" id="emailTrackingCodeBtn"
```

Comment:
This is what the mail looks for me here: 
![image](https://user-images.githubusercontent.com/273120/62021002-5a466480-b219-11e9-80d2-4c7fe829a349.png)

There seem to be some escaped HTML elements, and some closing `</p>` elements visible. 

We should probably remove the link `You can customise this tracking code at https://...` because the developer might not have access to it? Could also leave it there though.

In general it be great if we could also mention the alternative tracking methods in all the different tracking code emails like:

```
....Here the text we already have in the mail....

------------------------------

** Further ways of tracking

*** Integrations
In most websites, blogs, CMS, etc. you can use a pre-made plugin to do the technical work for you:
https://matomo.org/integrate/

*** Log Analytics
If the JavaScript tracking method isn’t feasible, you can use server log analytics as an alternative method for tracking your website’s users.
https://matomo.org/log-analytics/

*** Mobile apps and SDKs
Not tracking a website? You can alternatively track a mobile app or any other type of application using one of the available SDKs.
https://matomo.org/integrate/#programming-language-platforms-and-frameworks

*** HTTP Tracking API
The HTTP Tracking API allows you to track anything. This may be useful if you are using a programming language for which no SDK exists yet. It may also be useful when you want to track devices or application in a special way.
https://developer.matomo.org/api-reference/tracking-api
```

To keep things simple let's maybe not use the Tag Manager for now since a developer would likely not have access to configure things anyway. We will instead need eventually a different email button for Tag Manager.

@katebutler  WhiteLabel has a setting to remove links named `removeLinksToMatomo`. We will need to test what happens if this is enabled. I suppose the links wouldn't be removed in this case since it is plain HTML. At the same time you can't know in core if this setting is enabled or not. We may need an event that WhiteLabel can listen to like:

```php
$showDifferentWaysOfTracking = true;
Piwik::postEvent('SitesManager.showDifferentWaysOfTrackingInTrackingCodeMail', array(&$showDifferentWaysOfTracking));
// then only include the different ways if `$showDifferentWaysOfTracking === true`
```